### PR TITLE
Update `cpu_info` to include `load_percentage` on windows

### PR DESCRIPTION
**Pull request by directionless**
It looks like windows tracks this, and there was an old PR (#5307) for the update. As the code was simple enough I've ported it forward. 

Co-authored-by: Cameron Haupt <chaupt2013@my.fit.edu>

### CODE REVIEW

File: osquery/tables/system/windows/cpu_info.cpp [hunk 1]
```diff
@@ -57,6 +57,9 @@ QueryData genCpuInfo(QueryContext& context) {
     (data.GetLong("MaxClockSpeed", number))
         ? r["max_clock_speed"] = INTEGER(number)
         : r["max_clock_speed"] = "-1";
+    (data.GetLong("LoadPercentage", number))
```
Comment (zwass) [inline osquery/tables/system/windows/cpu_info.cpp:60]: This ternary ends up returning `-1` when the actual load percentage reported is `0`.

Comment (zwass) [reply osquery/tables/system/windows/cpu_info.cpp:60]: No, actually that doesn't seem to be the case as I've now seen it successfully return `0`. Not clear exactly why it returns `-1` sometimes.

Comment (zwass) [reply osquery/tables/system/windows/cpu_info.cpp:60]: Okay I manually ran the WMI query and verified that it sometimes just doesn't return a value.

```
Get-WmiObject -Query 'Select LoadPercentage from Win32_Processor'
```

**Review by zwass — APPROVED** [review]
LGTM

File: osquery/tables/system/windows/cpu_info.cpp (hunk 1 above)
Comment (directionless) [reply osquery/tables/system/windows/cpu_info.cpp:60]: I suspect sometimes WMI is locked or weird. It all feels like heavy RPC stuff

**Pull request merged**